app/controllers/actions/AuthAction.scala

```diff
@@ -165,15 +165,10 @@ class FullAuthentication @Inject()(override val authConnector: AuthConnector,
 
   private def redirectToInterceptPages[A](enrolments: Enrolments, request: Request[A],
```

Comment:
Would it not be better if this returned an Option now instead of an Either? It's returning either no value at all (Unit) or a value.

Reply:
fixed